types.ts

```diff
@@ -859,6 +864,20 @@ export type Authentication =
   | GoogleServiceAccountAuthentication
   | CustomAuthentication;
 
+/**
+ * Exported to avoid Typescript not-used error
+ * @hidden
+ */
+export type _AllowedAuthenticationExtendsBaseAuthenticationCheck = EnsureExtends<
```

Comment:
This is fine but I think you could also put this in a test file and get the same effect (and api_test.ts already has some example of expressions meant TS verification that don't do anything at runtime).

Reply:
Done